Header header

uint32 width
uint32 height
uint8[] bgr
float32[] points
int32[] label
ObjectData[] objects
PlaneData[] planes
